.github/workflows/ods.yml

```diff
@@ -620,7 +628,7 @@ jobs:
           if [[ "${{ github.ref }}" != refs/tags/* ]]
           then
             UI_VERSION=${UI_VERSION}-$(git rev-parse --short HEAD)
-          fi
+          figit
```

Comment:
I have never heard of `if ... figit`. Is this a typo, or does it have a special meaning?

Reply:
Thx good catch! Came from my terminal dropdown that was too slow to take my keys :/